### ISSUE TITLE
backport CVE-2019-11358 fixes to jquery 1.x and 2.x

### ISSUE THREAD

**Issue by hannob** [jquery/jquery#4559]
jquery 3.4.0 made a change that in some cases prevents security issues, which is tracked as security vulnerability CVE-2019-11358.

Many web applications include versions of jquery 1.x and 2.x, which haven't received this change, because updating major versions is difficult. While backport patches are available, I think it would be very helpful to have official releases with these changes.

I'm primarily asking this because I've seen people getting bugreports because they bundle a component that has an unfixed CVE. This also may lead to warnings by security tools.

**Comment by mgol** [comment]
Our current release process is quite different to one we had in jQuery 1.x/2.x days so if we decided to release such updates we'd have to do a lot of it by hand. That involves building jQuery for Bower/npm, updating the jquery-dist repo, including its README, building it again for a few CDNs we publish to, publish to npm, etc.

And even if we did that, there's another issue that we won't be able to fix in 1.x/2.x as that involves a breaking change: https://github.com/advisories/GHSA-rmxg-73gg-4p98/. Even with such patch releases, having a secure jQuery unfortunately would still require updating to version `>=3.0.0` or parching the file manually; your GitHub reports wouldn't get zeroed out without such updates.

I know WordPress decided to patch jQuery on its own with both patches, its modified jQuery copy is included in WordPress 5.3: https://core.svn.wordpress.org/tags/5.3/wp-includes/js/jquery/
